3rdparty/haskell/c2hs-package.bzl

```diff
@@ -0,0 +1,522 @@
+package = (
```

Comment:
I assume that file was generated using `cabal2bazel` and then modified manually?

Could you highlight where it was modified in a comment, so it's easy to see if we can go back to generated at some point in the future?

Reply:
Indeed. PTAL.

Reply:
Thanks! LGTM